standards/code-style.adoc

```diff
@@ -722,56 +726,57 @@ When multiple methods are chained in the same line - for example when using
 Builders - every call to a method should go in its own line, breaking the line
 before the `.`
 
-[source, java]
+[source, kotlin]
 ----
-Picasso.with(context).load("https://i.imgur.com/kG0deA5.jpg").into(imageView);
+Picasso.with(context).load("https://i.imgur.com/kG0deA5.jpg").into(imageView)
 ----
 
-[source, java]
+[source, kotlin]
 ----
 Picasso.with(context)
         .load("https://i.imgur.com/kG0deA5.jpg")
-        .into(imageView);
+        .into(imageView)
 ----
 
 __Long parameters case__
 
 When a method has many parameters or its parameters are very long, we should
 break the line after every comma `,`
 
-[source, java]
+[source, kotlin]
 ----
-loadPicture(context, "https://i.imgur.com/kG0deA5.jpg", imageViewProfilePicture, clickListener, "Title of the picture");
+loadPicture(context, "https://i.imgur.com/kG0deA5.jpg", imageViewProfilePicture, clickListener, "Title of the picture")
 ----
 
-[source, java]
+[source, kotlin]
 ----
 loadPicture(context,
         "https://i.imgur.com/kG0deA5.jpg",
         imageViewProfilePicture,
         clickListener,
-        "Title of the picture");
+        "Title of the picture")
 ----
 
 === RxJava chains styling
 
 Rx chains of operators require line-wrapping. Every operator must go in a new line and the line should be broken before the `.`
 
-[source, java]
+[source, kotlin]
 ----
-public Observable<Location> syncLocations() {
+fun syncLocations(): Observable<Location> {
     return databaseManager.getAllLocations()
-            .concatMap(new Func1<Location, Observable<? extends Location>>() {
-                @Override public Observable<? extends Location> call(Location location) {
-                    return retrofitService.getLocation(location.id);
+            .concatMap(object : Func1<Location, Observable<out Location>>() {
```

Comment:
I don't think we need Func1 in Kotlin here? That's a crutch since we didn't have higher order functions. This can probably just be `concatMap{ retrofitService.getLocation(location.id) }`